source/features/user-local-time.tsx

```diff
@@ -80,9 +80,9 @@ function parseOffset(date: string): number {
 function init(): void {
 	const hovercard = select('.js-hovercard-content > .Popover-message')!;
 
-	observeElement(hovercard, async () => {
+	observeElement(hovercard, async ([mutation]) => {
 		if (
-			select.exists('.rgh-local-user-time', hovercard) || // Time already added
```

Comment:
Dropped this condition since it's always false (the contents of the hovercard element are erased every time it disappears).